### DISCUSSION TITLE
How does the folder addon save its folder data?

### DISCUSSION THREAD

**Discussion by Platformer444** [ScratchAddons/ScratchAddons#4846] [Help] [answered]
I want to know how the folder addon saves the folder data. I am trying it for days but can't find it.

**Comment by GarboMuffin** [comment, marked as answer]
It's stored in the name of the sprite. A sprite named "xyz" in the folder "abc" actually has the name "abc//xyz".

I would imagine there's a reason you're asking this. Why not elaborate on that?

**Reply by Platformer444** [reply]
So the <folder_name>// prefix is enough for the saving and loading work?
